CMakeLists.txt

```diff
@@ -0,0 +1,122 @@
+# CMakeList.txt
+#
+# Copyright (C) 2006-2020 wolfSSL Inc.
+#
+# This file is part of wolfSSL. (formerly known as CyaSSL)
+#
+#
+
+cmake_minimum_required (VERSION 2.6)
+
+####################################################
+# Project
+####################################################
+project(wolfssl)
+find_package (Threads)
+
+####################################################
+# Compiler
+####################################################
+set(CMAKE_C_COMPILER gcc)
+set(CMAKE_CXX_COMPILER gcc)
```

Comment:
This can cause build problem when making C++ mobile project that has package manager. something like conan.

Reply:
Agree. Omit these and let cmake use the default.